litellm/src/rustic_ai/litellm/conf.py

```diff
@@ -125,6 +125,11 @@ class LiteLLMConf(BaseAgentProps):
     If set to 0, the agent will not retry on tool parse error.
     """
 
+    vertex_location: Optional[str] = None
```

Comment:
We should add `vertex_project` too as that is also listed here - 
https://docs.litellm.ai/docs/providers/vertex#dynamic-params